scw/config.go

```diff
@@ -185,7 +188,22 @@ func MustLoadConfig() *Config {
 
 // LoadConfig read the config from the default path.
 func LoadConfig() (*Config, error) {
-	return LoadConfigFromPath(GetConfigPath())
+	configPath := GetConfigPath()
+	cfg, err := LoadConfigFromPath(configPath)
+
+	var configNotFoundError *ConfigFileNotFoundError
+	if err != nil && goerrors.As(err, &configNotFoundError) {
+		if strings.HasSuffix(configPath, ".yaml") {
+			configPath = strings.TrimSuffix(configPath, ".yaml") + ".yml"
+		}
```

Comment:
File reading will be duplicated on fail
```suggestion
	if err != nil && goerrors.As(err, &configNotFoundError) && strings.HasSuffix(configPath, ".yaml") {
		configPath = strings.TrimSuffix(configPath, ".yaml") + ".yml"
```